packages/core/src/lib/wait_for_remote_peer.ts

```diff
@@ -76,12 +85,13 @@ export async function waitForRemotePeer(
   }
 }
 
+//TODO: move this function within protocol SDK class
```

Comment:
Let's link this `TODO` to an issue or epic.

Reply:
done: c66ad587b2c212688b763655ca5cf4157fb2b0e2